src/sentry/static/sentry/app/components/events/meta/metaProxy.tsx

```diff
@@ -52,23 +58,30 @@ export class MetaProxy {
   }
 }
 
-export function withMeta(event) {
+export function withMeta<T>(event: T): T {
   if (!event) {
-    return null;
+    return event;
   }
 
   // Return unproxied `event` if browser does not support `Proxy`
   if (typeof window.Proxy === 'undefined' || typeof window.Reflect === 'undefined') {
```

Comment:
can't it be like `!window.Proxy` ?

Reply:
I really don't want to do this.

Reply:
just looking back through the review and saw my low-effort reply: this metaprogramming code used to have a bug where the distinction between `undefined` and `null` mattered, and we certainly don't want to treat `0` like either of them. So let's keep it explicit for now.